time.cabal

```diff
@@ -151,3 +150,66 @@ test-suite tests
         Test.AddDays
         Test.AddDaysRef
         Test.TestUtil
+        
+benchmark bench
+    hs-source-dirs: lib, benchmark
```

Comment:
Do you need to rebuild all of `lib`? I think you can remove this and all the modules in lib from the `other-modules` field.